### DISCUSSION TITLE
why does chat-13B endless self talking

### DISCUSSION THREAD

**Discussion by sunyuhan19981208** [ggml-org/llama.cpp#1475] [Q&A] [answered]
After I ask him a long question in Chinese, he began talking to himself endless. Have you ever met the same thing? 
![截屏2023-05-16 上午11 56 48](https://github.com/ggerganov/llama.cpp/assets/50269851/f199085d-d3d4-48f2-a48c-35d1d2449c59)

**Comment by SlyEcho** [comment, marked as answer]
It happens sometimes, you can try a lower temperature like 0.1

**Comment by bugbountyguy** [comment]
My bot did the exact same thing with 13B. Asked it "How old is your model" and it just spat out nonstop self-conversations (Not to mention, didn't reply with "Bot:" for the prompt like I set, which does work for 30B). I even had it switch languages randomly at one point to Korean - didn't know what the hell was going on. Tried to lower the temperature from 0.7 to 0.1, didn't change a thing. Any other settings that can be lowered? Or maybe it's just the model? Using `ggml-vic13b-uncensored-q4_2.bin`

![image](https://github.com/ggerganov/llama.cpp/assets/88092414/2eee3eeb-11d7-4e55-bb48-f549926d265f)
